Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Melee/bat_home_run.yml

```diff
@@ -0,0 +1,38 @@
+# DeltaV "Home Run Bat"
+
+- type: entity
+  name: home run bat
+  parent: BaseItem
+  id: BaseBallBatHomerun
+  description: Heavy metal bat with an extra kick.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Melee/home_run_bat.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 35
```

Comment:
We need to take a look at toning these numbers back. I don't think that this should be a solid argument for the best melee weapon in the syndicate. It would be better off being balanced around paying for the knockback utility than for raw damage output.

Reply:
i wrote those damage values when sleep deprived my b

Reply:
it's been toned down significantly